streams/src/commonMain/kotlin/com/mirego/trikot/streams/reactive/PublishSubjectImpl.kt

```diff
@@ -110,8 +109,10 @@ open class PublishSubjectImpl<T> : PublishSubject<T> {
     }
 
     override fun complete() {
-        isCompleted.setOrThrow(false, true)
-        subscriptions.value.forEach { it.dispatchCompleted() }
+        serialQueue.dispatch {
```

Comment:
Not sure if this is an issue, but this block can be executed after `onNoSubscription()` has been called.

Reply:
Not an issue, if there is no more subscriber, no one will receive the completion()